server/utils/Random.ts

```diff
@@ -0,0 +1,5 @@
+export default class Random {
+    static getRandomNumber(min: number, max: number): number {
+        return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
Can you add a comment that this generates a number in [min, max] inclusive of both endpoints?